### PR TITLE
studyJob with non-kubeflow namespace cannot work

### PR DESCRIPTION
**Pull request by hougangliu**
Fixes: #301

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/katib/302)

### CODE REVIEW

**Comment by hougangliu** [comment]
Before create a non-kubeflow (saying `nonkubeflow`) namespace studyJob, we should doc that user must run below command:

```
$ cat mcrbac.yaml
apiVersion: v1
kind: ServiceAccount
metadata:
  name: metrics-collector
  namespace: nonkubeflow
---
kind: ClusterRoleBinding
apiVersion: rbac.authorization.k8s.io/v1
metadata:
  name: metrics-collector
  namespace: nonkubeflow
roleRef:
  apiGroup: rbac.authorization.k8s.io
  kind: ClusterRole
  name: metrics-collector
subjects:
- kind: ServiceAccount
  name: metrics-collector
  namespace: nonkubeflow
$ kubeflow apply -f  mcrbac.yaml
```

File: pkg/types.go [hunk 1]
```diff
@@ -1,5 +1,5 @@
 package pkg
 
 const (
-	ManagerAddr = "vizier-core:6789"
+	ManagerAddr = "vizier-core.kubeflow:6789"
```
Comment (johnugeorge) [inline pkg/types.go]: Why is this change added ?

Comment (hougangliu) [reply pkg/types.go]: kube-dns will assign k8s services a DNS A record for a name of the form my-svc.my-namespace.svc.cluster.local and  my-svc.my-namespace. This resolves to the cluster IP of the Service.
vizier-core is service name, and if we don't add namespace, when Pod in non-kubeflow namespace cannot resolve vizier-core to its cluster IP (it tries to resolve vizier-core.non-kubeflow).

Comment (johnugeorge) [reply pkg/types.go]: Yes. I mean to say that vizier core can run in any namespace
